### PR TITLE
Issue 220 - First simple prototype for list view

### PR DESCRIPTION
**Pull request by asimonw**
A few routes to get us going.
The naming of the routes still to TBC. Share your thoughts.

I'm using Bulma for the CSS. It might be a good option for quick prototyping. Not 100% support for IE10-11 though, but modern and fairly lightweight. Currently just pulled in through a CDN, but if we decide to use it more seriously, I'd use npm and work with it on the SASS level.

### CODE REVIEW

**Comment by asimonw** [comment]
Start at `/list`

**Comment by MertonL** [comment]
Tested and it's a nice MVP!
Assets load quickly (tested with ~1000 assets and the performance is still fine), looking at the code bulma seems like a useful tool to use. Nice modular code as well 👍 

Small bug I noticed is that if an Asset has multiple 2nd level tags from the same 1st level tag, then that Asset will be displayed multiple times for each 2nd level tag.

**Ex:** 
An asset with these tags: 
screen shot 2018-02-21 at 13 15 28

Will display like this:
screen shot 2018-02-21 at 13 15 17

File: asset_manager/user_interface/views.py [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+
+def list(request):
+	tagGroups = TagGroup.objects.all()
+	context = { 'tagGroups': tagGroups }
+	return render(request, 'user_interface/list.html', context)
+
+def tag_group(request, tag_group_id):
+	tagGroup = TagGroup.objects.get(id=tag_group_id)
+	subTags = Tag.objects.filter(group__id=tag_group_id)
+	assets = Asset.objects.filter(tags__in=subTags)
```
Comment (MertonL) [inline asset_manager/user_interface/views.py]: To fix the duplication issue, adding `.distinct()` to the end of this line seems to resolve the issue.

Comment (asimonw) [reply asset_manager/user_interface/views.py]: Nice spot @MertonL and looks like a good fix. Are you going to commit the fix? Happy to do it too.

Comment (MertonL) [reply asset_manager/user_interface/views.py]: Commited!

**Pull request merged**